core/domain/event_services.py

```diff
@@ -65,6 +65,75 @@ def record(cls, *args, **kwargs):
         cls._handle_event(*args, **kwargs)
 
 
+class StatsEventsHandler(BaseEventHandler):
+    """Event handler for incremental update of analytics model using batched
+    event models.
+    """
+
+    EVENT_TYPE = feconf.EVENT_TYPE_ALL_STATS
+
+    @classmethod
+    def _handle_event(cls, exploration_id, exp_version, event_dicts):
+        event_params = []
```

Comment:
This is not so great, it's a list of dicts of all sorts of formats and it's a bit hard to work with. Can we just pass to update_stats a single batched dict with fixed key/value definitions, maybe something like:

    {
        new_start_events: 5,
        new_complete_events: 3,
        ...
    }

i.e. the minimal amount of info needed for the update function to do its job? That way you can actually document the dict and its keys/values with certainty, and you aren't passing a whole lot of extra stuff over the wire as well.

Reply:
@seanlip That sounds right. I've made the changes.